docs/src/components/Nav/NavLinkNext.js

```diff
@@ -7,17 +7,15 @@ const propTypes = {
 };
 
 // This component is needed as a compatibility layer inside of AdminStructure.Nav due to the odd way that
-// Link works inside of `next/link` where it does not render an element itself. This component allows us to 
+// Link works inside of `next/link` where it does not render an element itself. This component allows us to
 // maintain a structure that is consistent with other link components, and direct props into the expected location.
 const NavLinkNext = ({ children, isActive, ...rest }) => {
   const classList = [rest.className, isActive ? 'active' : undefined];
 
   return (
-    <div className={classList.join(' ')}>
-      <Link {...rest}>
-        <a>{children}</a>
-      </Link>
-    </div>
+    <Link {...rest}>
+      <a className={classList.join(' ')}>{children}</a>
+    </Link>
```

Comment:
Turns out we were getting an extra div that was not needed structurally speaking and it was making the clickable area only surround the text instead of the whole button. This switch resolves things. This compatibility component is probably a good idea to have for now, mainly due to how we are doing things with the classList.